### PR TITLE
Configuration files restructuring

### PR DESCRIPTION
**Pull request by aegroto**
This patch modifies the configuration system to load different files and then build a unified map.

The test cases added verify that the final map is equal to the one generated by the old monolithic code.

### CODE REVIEW

**Review by drendog — APPROVED** [review]

**Comment by Helias** [comment]
LGTM, need tests

File: module/config.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import yaml, os
+
+def load_configurations(path='config/'):
```
[the remark this answers is not in the document]
Comment (aegroto) [reply module/config.py:3]: I would avoid using a global variable. Caching the value using an object would make the trick though, I'd need to check if there's some standard in Python to lazy-initialize a value.

**Comment by Helias** [comment]
is this in production?

**Comment by Herbrant** [comment]
It's now in production.

**Pull request merged**